src/voice-pe/customizing/disassemble.md

```diff
@@ -12,11 +12,18 @@ Follow these steps if you have 3D-printed your own enclosure parts and now want
 
 ## Prerequisites
 
+{% steps %}
+{% step " " " " %}
```

Comment:
This should be updated to the new code.

Reply:
https://github.com/NabuCasa/support/pull/519